Tools: Add link to provider online check (#169)
oasis-open/csaf-documentation: content/tools.md

{{% card %}}
### [DependencyTrack CSAF Integration](https://github.com/csaf-sbom/hyades-csaf)
An integration for the popular dependency management system DependencyTrack.
{{% /card %}}

{{% card %}}
### [CSAF Provider Online Check](https://check.provider.csaf.dev/)

> A website which provides easy access to CSAF Checker and Validation tools.

Reviewer: If I read "validation", it sounds like "CSAF validator" which is a separate tool...
```suggestion
A website which provides easy access to the CSAF Provider Checker.
```
